common/src/main/java/org/ergoplatform/appkit/ErgoValue.java

```diff
@@ -73,8 +77,8 @@ public boolean equals(Object obj) {
         return new ErgoValue(Iso.jshortToShort().to(Short.valueOf(value)), ErgoType.shortType());
     }
 
-    static public ErgoValue<scala.Int> of(int value) {
-        return new ErgoValue(Iso.jintToInt().to(Integer.valueOf(value)), ErgoType.integerType());
+    static public ErgoValue<Integer> of(int value) {
+        return new ErgoValue(Iso.jintToInt().to(value), ErgoType.integerType());
```

Comment:
Oh, no. If change this, then other types (short, byte etc) should be changed. I'm not sure that the change is needed, but if it is, then all types should be changed atomically.

Reply:
It is needed because the actual value is an Integer (Java) although there is the Iso.jintToInt() call. It somehow always is converted to Java Integer on the fly. I don't know if this is the same for the other types, if yes it should be changed in a separate PR. This change here was necessary for the correct declaration of `getErgoValue` above.